src/components/Users/UserListAndCard.tsx

```diff
@@ -81,9 +88,21 @@ const getNameAndStatusCard = (
       <div className="flex flex-row justify-between gap-x-3">
         <div className="flex flex-col">
           <div className="flex items-center gap-x-3">
-            <h1 id={`name-${user.username}`} className="text-base font-bold">
-              {formatName(user)}
-            </h1>
+            <TooltipProvider delayDuration={200}>
+              <Tooltip>
+                <TooltipTrigger asChild>
+                  <h1
+                    id={`name-${user.username}`}
+                    className="text-base font-bold truncate max-w-[7rem]"
+                  >
+                    {formatName(user)}
+                  </h1>
+                </TooltipTrigger>
+                <TooltipContent>
```

Comment:
I'd suggest a bit more styling to the tooltip content. Check the tooltip for Occupancy Badge in Facility for instance:

<img width="558" alt="image" src="https://github.com/user-attachments/assets/4e38320c-83c3-4866-9149-15525dc0a5b1" />

Reply:
Okay, I just added more styling and dynamic transitions on state and side.